### ISSUE TITLE
Predict sentiment of input text

### ISSUE THREAD

**Issue by ibrahimsharaf** [ibrahimsharaf/doc2vec#3]
The script now only supports training and testing given an input dataset, we need to add a new function to support prediction given a new example.

- Save the trained models (doc2vec, random forest/logisitc regression)
- Load them when the user wants to predict the sentiment of a specific input text.

Depends on #14

**Comment by avinashsai** [comment]
@ibrahimsharaf what should be done in the next step??

**Comment by ibrahimsharaf** [comment]
@avinashsai a PR including the first two issues in this previous PR (https://github.com/ibrahimsharaf/doc2vec/pull/6)?

**Comment by avinashsai** [comment]
Sure, i will open another PR with the first two.

**Comment by luciferldy** [comment]
已收到，谢谢
